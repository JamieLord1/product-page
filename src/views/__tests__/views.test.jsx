import React from 'react';
import renderer from 'react-test-renderer';
import Views from '../views'

it('renders correctly', () => {
	const tree = renderer.create(<Views />).toJSON();
	expect(tree).toMatchSnapshot();
});
