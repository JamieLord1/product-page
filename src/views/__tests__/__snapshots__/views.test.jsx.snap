// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div
      className="product-page-colour-filter ant-select ant-select-enabled"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      onMouseOut={[Function]}
      onMouseUp={[Function]}
      onTouchStart={[Function]}
    >
      <div
        aria-autocomplete="list"
        aria-controls="test-uuid"
        aria-expanded={false}
        aria-haspopup="true"
        className="ant-select-selection
            ant-select-selection--multiple"
        data-testid="color-filter"
        role="combobox"
      >
        <div
          className="ant-select-selection__rendered"
        >
          <div
            className="ant-select-selection__placeholder"
            onClick={[Function]}
            onMouseDown={[Function]}
            style={
              Object {
                "WebkitUserSelect": "none",
                "display": "block",
                "userSelect": "none",
              }
            }
            unselectable="on"
          >
            Colour filter
          </div>
          <ul>
            <li
              className="ant-select-search ant-select-search--inline"
            >
              <div
                className="ant-select-search__field__wrap"
              >
                <input
                  autoComplete="off"
                  className="ant-select-search__field"
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  value=""
                />
                <span
                  className="ant-select-search__field__mirror"
                >
                  
                   
                </span>
              </div>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="product-page-list-container"
    >
      <h2>
        Loading...
      </h2>
    </div>
    <hr />
    <h2
      className="product-page-total"
    >
      £ Total 0.00
    </h2>
  </div>
</div>
`;
